app/views/artists/appointments/_edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<h4>Edit Appointment</h4>
+<%= form_tag "/users/appointments/#{appointment.id}", method: :patch do %>
```

Comment:
don't think your search for user picked up user  in the strings, think this can be changed to 

`<%= form_tag "artists_appointment_path(appointment.id)", method: :patch do %>`

Reply:
I think I forgot to check master after we merged your PR today, so I missed all these. Thanks for catching them!